AllReadyApp/Web-App/AllReady/Areas/Admin/Features/CampaignManagerInvites/CreateCampaignManagerInviteCommandHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using AllReady.Models;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Threading.Tasks;
+
+namespace AllReady.Areas.Admin.Features.CampaignManagerInvites
+{
+    public class CreateCampaignManagerInviteCommandHandler : AsyncRequestHandler<CreateCampaignManagerInviteCommand>
+    {
+        private AllReadyContext _context;
+
+        public CreateCampaignManagerInviteCommandHandler(AllReadyContext context)
+        {
+            _context = context;
+        }
+
+        public Func<DateTime> DateTimeUtcNow = () => DateTime.UtcNow;
+
+        protected override async Task HandleCore(CreateCampaignManagerInviteCommand message)
+        {
+            var campaign = await _context.Campaigns.AsNoTracking().SingleOrDefaultAsync(c => c.Id == message.Invite.CampaignId);
+            if (campaign == null) throw new ArgumentException("CampaignId cannot be null for campaign manager invite");
```

Comment:
remove these two lines. We don't do this type of validation in command handlers. We should validate in the controller before we send the command.